src/app/App.js

```diff
@@ -1,27 +1,15 @@
 import "./App.css";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import { MovieListPage } from "../pages/MovieListPage/MoviesListPage.jsx";
-import { MovieDetails } from "../entities/MovieDetails/MovieDetails.jsx";
-import { SearchForm } from "../entities/SearchForm/SearchForm.jsx";
-import { loader as movieLoader } from "../routes/movie.jsx";
-import { CustomButton } from "../shared/ui/CustomButton/CustomButton.jsx";
-import { SearchFormWrapper } from "../entities/SearchFormWrapper/SearchFormWrapper.jsx";
+import { moviecrudPaths } from "./paths/Moviecrud.js";
 
 
 const router = createBrowserRouter([
   {
     path: "/",
     element: <MovieListPage />,
     children: [
-      {
-        path: "/",
-        element: <SearchFormWrapper/>
-      },
-      {
-        path: "/:movieId",
-        loader: movieLoader,
-        element: <MovieDetails />,
-      },
+      moviecrudPaths
```

Comment:
I didn't mean to separate the whole object, I meant only paths, like:
```
const paths = {
home: '/',
movieDetails: '/:movieId', 
...
```
}

Reply:
done